BDArmory/UI/BDArmorySetup.cs

```diff
@@ -3313,7 +3313,17 @@ void WindowSettings(int windowID)
                 {
                     if (BDArmorySettings.RUNWAY_PROJECT != (BDArmorySettings.RUNWAY_PROJECT = GUI.Toggle(SLeftRect(++line), BDArmorySettings.RUNWAY_PROJECT, StringUtils.Localize("#LOC_BDArmory_Settings_RunwayProject"))))//Runway Project
                     {
-                        if (HighLogic.LoadedSceneIsEditor && EditorLogic.fetch.ship is not null) GameEvents.onEditorShipModified.Fire(EditorLogic.fetch.ship);
+                        if (HighLogic.LoadedSceneIsEditor && EditorLogic.fetch.ship is not null) GameEvents.onEditorShipModified.Fire(EditorLogic.fetch.ship); if (oldSpaceHacks != BDArmorySettings.SPACE_HACKS)
+
+                            SpawnUtils.HackActuatorsOnNewVessels(BDArmorySettings.RUNWAY_PROJECT);
+                            if (BDArmorySettings.RUNWAY_PROJECT) // Add the hack to all in-game intakes.
+                            {
+                                foreach (var vessel in BDATargetManager.LoadedVessels)
+                                {
+                                    if (vessel == null || !vessel.loaded) continue;
+                                    SpawnUtils.HackActuators(vessel, true);
```

Comment:
This ought to be called as `SpawnUtils.HackActuators(vessel, BDArmorySettings.RUNWAY_PROJECT);` and not be guarded by `if (BDArmorySettings.RUNWAY_PROJECT) // Add the hack to all in-game intakes.` so that disabling RWP reverts the hack.